TestCases/compliance-level-3/0081-feel-getentries-function/0081-feel-getentries-function.dmn

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0081-feel-getentries-function" name="0081-feel-getentries-function" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <description>FEEL built-in function 'get entries(m)' in unspecified category</description>
+
+    <decision name="decision_001" id="_decision_001">
+        <description>Tests FEEL expression: 'get entries()' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get entries()'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="list" name="decision_001"/>
```

Comment:
The spec DMNv1.2 says:
![image](https://user-images.githubusercontent.com/1699252/51470603-aac59c80-1d74-11e9-9e8f-b7c858c95d18.png)

then further specifically for InformationItem:
![image](https://user-images.githubusercontent.com/1699252/51470700-f8daa000-1d74-11e9-8281-2b50490fb4d6.png)

hence I derive `list` is not allowed to be used as a built-in type in typeRef (moreover IIRC there is currently undergoing discussion to make it supports generics for next DMN version)

either this DMN model needs to specify an ItemDefinition name `list` or would be best to drop the typeRef in this case(s)